optimum/onnxruntime/utils.py

```diff
@@ -218,6 +218,28 @@ def validate_provider_availability(provider: str):
         )
 
 
+def possible_io_binding_for_provider(providers: List[str]) -> bool:
```

Comment:
IO binding is only for CUDAExecutionProvider, so do we need this involved logic?

Reply:
If we decide whether to use io binding or not by model.get_provider(), there will be cuda ep in providers when using tensorRT.